lib/iris/tests/unit/coords/test_Cell.py

```diff
@@ -87,29 +86,31 @@ def test_len_1_numpy_array(self):
 
 class Test___eq__(tests.IrisTest):
     def test_datetimelike(self):
-        # Check that cell equality works with objects with a "timetuple".
-        dt = mock.Mock(timetuple=mock.Mock())
-        cell = mock.MagicMock(
-            spec=Cell, point=datetime.datetime(2010, 3, 21), bound=None
+        # Check that cell equality works with different datetime objects
+        # using the same calendar
+        point = cftime.datetime(2010, 1, 1, calendar="gregorian")
+        cell = Cell(
+            datetime.datetime(2010, 1, 1),
+            bound=[
+                datetime.datetime(2010, 1, 1),
+                datetime.datetime(2011, 1, 1),
+            ],
```

Comment:
```suggestion
            bound=None,
```

Otherwise this changes what is under test.

Reply:
Done